components/sup/src/manager/service/terminator.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2019 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::spawned_future::SpawnedFuture;
+use crate::sys::{service, ShutdownMethod};
+use futures::sync::oneshot;
+use habitat_core::os::process::Pid;
```

Comment:
```suggestion
use habitat_core::{os::process::Pid, outputln};
```
It's confusing where `outputln` comes from. At some point I want to fix them all and remove the `#[macro_use`] [here](https://github.com/habitat-sh/habitat/blob/master/components/sup/src/lib.rs#L58), but being explicit in the meantime makes the code easier to understand and reduces the work of that later change.

I imagine this applies elsewhere as well, but I'll just make this comment once.

Reply:
Good call... I'd also like to clean up `lib.rs` to ultimately ditch as many of the `extern crate` calls as we can. This'll be a good prelude to that.